code/obj/item/cigarette.dm

```diff
@@ -527,14 +527,20 @@
 	item_state = "cigpacket"
 	w_class = W_CLASS_TINY
 	throwforce = 2
-	var/cigcount = 6
+	var/max_cigs = 6
 	var/cigtype = /obj/item/clothing/mask/cigarette
 	var/package_style = "cigpacket"
 	flags = ONBELT | TABLEPASS | FPRINT
 	stamina_damage = 3
 	stamina_cost = 3
 	rand_pos = 1
 
+	New()
+		..()
+		for(var/i in 1 to src.max_cigs)
+			src.contents += new src.cigtype(src)
```

Comment:
```suggestion
			new src.cigtype(src)
```
The 'contents' list is automatically managed. If you move or create something in the packet it will automatically be in the `contents` list. Here the first argument `src` means that you are creating the cigarette in the packet.